### PR TITLE
Clarified note on JSON over HTTP

### PR DESCRIPTION
**Pull request by mitsuhiko**
Refs https://github.com/source-map/source-map-rfc/issues/34

### CODE REVIEW

File: source-map.bs [hunk 1]
```diff
@@ -415,6 +415,18 @@ JavaScript style single line comments.
...
-<mark>For historic security reasons, when delivering source maps over HTTP, servers may prepend a
-line starting with the string `)]}'` to the source map.  If the response starts with this
-string clients must ignore the first line.</mark>
+For historic reasons, when delivering source maps over HTTP, servers may prepend a line
+starting with the string `)]}'` to the source map.  A client fetching a source map via
+HTTP thus should check if the response starts with this string and then ignore the
+first line.
+
+```
+)]}garbage here
```
Comment (nicolo-ribaudo) [inline source-map.bs:424]:
```suggestion
)]}'garbage here
```

The prefix should include ', right?

**Comment by mitsuhiko** [comment]
@nicolo-ribaudo indeed. Funny enough though Chrome and Safari are also happy with just `)]}` as prefix, but not Firefox.

**Review by littledan — APPROVED** [review]
We can integrate this with whatever `fetch` text we add later; this is good for now. @nicolo-ribaudo

**Review by nicolo-ribaudo — APPROVED** [review]
I agree -- @mitsuhiko if you want to work on using the fetch infrastructure and you need any help, feel free to ask to me!

**Pull request merged**